src/store/index.ts

```diff
@@ -144,10 +150,14 @@ export function getFSXAModule<R extends RootState>(
             getFSXAConfiguration(this.state.fsxa.configuration),
             this.state.fsxa.configuration.logLevel,
           );
-          let navigationData = await fsxaAPI.fetchNavigation(
-            path || null,
-            payload.defaultLocale,
-          );
+
+          let navigationData = await fsxaAPI
+            .fetchNavigation(path || null, payload.defaultLocale)
+            .catch(reason => {
+              if (isNotFoundError(reason)) return null;
```

Comment:
Can be replaced by `if (reason?.message === FSXAApiErrors.NOT_FOUND)`

Reply:
Why do we want to make the navigationData null in the first place? 
In line 162 we will refetch the navigation.

Reply:
> Can be replaced by `if (reason?.message === FSXAApiErrors.NOT_FOUND)`

It is possible that `reason` is e.g. a String and this results in a Syntax Error for the optional chaining operator.


> Why do we want to make the navigationData null in the first place? In line 162 we will refetch the navigation.

We need a falsy return value here to trigger line 162, `null` is most readable.